libqtile/widget/battery.py

```diff
@@ -496,9 +497,14 @@ def update(self) -> None:
         icon = self._get_icon_key(status)
         if icon != self.current_icon:
             self.current_icon = icon
-            self.draw()
+            self.draw(status)
 
-    def draw(self) -> None:
+    def draw(self,status: BatteryStatus) -> None:
+        if self.layout is not None:
+            if status.state == BatteryState.DISCHARGING and status.percent < self.low_percentage:
+                self.background = self.low_background
+            else:
+                self.layout.colour = self.background
```

Comment:
Why are you setting `self.background` in line 505 but `self.layout.colour` here?

Also, you've not got a line to reset `self.background` to a different value, once it's changed to `low_background` you can't change it back.